### DISCUSSION TITLE
Plus d'emplacements de missions secondaires, moyen d'en obtenir plus ou d'en supprimer

### DISCUSSION THREAD

**Discussion by Chevalier2** [Crownicles/Crownicles#1060] [Suggestions fermées]
Comme dit dans le titre, il serait je pense intéressant d'ajouter des emplacements pour les missions secondaires, actuellement limités à 3. Car parfois, surtout pour les joueurs actifs, on peut être bloqué avec 3 missions secondaires, parce que certaines sont compliquées/longues à réaliser ou tout simplement pas possible pour le joueur (ou qu'il n'en a pas envie). 

J'ai pensé à 3 solutions : 
- Ajouter des emplacements.
- Ajouter un moyen d'obtenir des emplacements.
- Ajouter un moyen de supprimer les missions secondaires avant leur expiration.

Ou, à la place des emplacements, avoir une réserve temporaire dans laquelle une mission ne peut rester que peu de temps avant d'être supprimée.

**Comment by Chevalier2** [comment]
Un exemple : 
![image](https://user-images.githubusercontent.com/72626953/160601636-3a30f978-ba8f-47bc-8c72-13e83121c6b0.png)

**Comment by romain22222** [comment]
je précise que la 3e suggestion est actuellement disponible dans le bot, suffit d'aller voir dans le shop des missions

**Comment by Chevalier2** [comment]
Mais coûte 10 gemmes, ce qui n'est pas du tout rentable. Ce que je voulais dire, c'est un moyen qui soit un minimum rentable. Les gemmes c'est trop rare pour en utiliser 10 pour ça. De toute façon, ce serait mieux d'avoir au moins un emplacement supplémentaire, même si c'est payant.

**Comment by BastLast** [comment]
Cette limitation a été mise en place par fair-play pour les joueurs casu, avoir 3 missions en même temps, c'est déjà pas mal.

Je met plutôt un veto sur les deux premières propositions, par contre il y aura surement un équilibrage sur le prix du passe mission.
